Source/stores.cpp

```diff
@@ -1022,7 +1023,7 @@ void StoreConfirm(Item &item)
 		prompt = _("Are you sure you want to repair this item?");
 		break;
 	default:
-		app_fatal(fmt::format("Unknown store dialog {}", static_cast<int>(stextshold)));
+		app_fatal(StrCat("Unknown store dialog ", static_cast<int>(stextshold)));

```

Comment:
This sort of use I think is an improvement, a single static string followed by some runtime value